src/main/java/com/savvato/tribeapp/repositories/PhraseRepository.java

```diff
@@ -2,12 +2,16 @@
 
 
 import com.savvato.tribeapp.entities.Phrase;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.Optional;
+
 @Repository
 public interface PhraseRepository extends CrudRepository<Phrase, Long> {
 
-
+    @Query(nativeQuery = true, value = "select * from phrase where id = ?")
+    Optional<Phrase> findPhraseByPhraseId(Long Id);
```

Comment:
should be findById()

Reply:
Will fix according to what we decide in Adverb conversation above.